CHANGES.md

```diff
@@ -1,5 +1,6 @@
 # Changes
 
+
```

Comment:
Added comment to CHANGES file, but since we're backwards compatible with the "old cache" I did not update the UPGRADING doc.
if the new cache config is not set in audit.yaml, then we fall back to what is in the cassandra.yaml (roles_validity_in_ms/roles_update_interval_in_ms/...)